mon-pix/app/services/competence-evaluation.js

```diff
@@ -0,0 +1,24 @@
+import Service, { inject as service }  from '@ember/service';
+
+export default class CompetenceEvaluationService extends Service {
+  @service store;
+  @service router;
+
+  async improve({ userId, competenceId, scorecardId }) {
+    try {
+      await this.store.queryRecord('competence-evaluation', { improve: true, userId, competenceId });
+      return this.router.transitionTo('competences.resume', competenceId);
+    } catch (error) {
+      const title = ('errors' in error) ? error.errors.get('firstObject').title : null;
+      if (title === 'ImproveCompetenceEvaluationForbidden') {
+        return this.reloadScorecard(scorecardId);
+      } else {
+        throw error;
+      }
+    }
+  }
+
+  async reloadScorecard(scorecardId) {
```

Comment:
Préfixer éventuellement cette méthode privée par `_` ? En particulier compte-tenu du fait qu'on est dans un service, ça me paraît important que son interface publique soit lisible.
```suggestion
  async _reloadScorecard(scorecardId) {
```